jobs/official_release.groovy

```diff
@@ -21,7 +21,8 @@ p.pipeline().with {
       'existing git ref(s) upon which to base the release. Eg. "w.9999.52"')
     stringParam('RELEASE_GIT_TAG', null,
       'git tag for the new release. Eg. "v888.0.0.rc1"')
-    stringParam('SPLENV_REF', scipipe.template.splenv_ref, 'conda env ref')
+    stringParam('SPLENV_REF', scipipe.template.splenv_ref, 'conda env ref or eups tag')
+    stringParam('RUBINENV_VER', scipipe.template.splenv_ref, 'rubin-env version')
```

Comment:
Are these both supposed to be splenv_ref, or should one be rubinenv_ver or some such?

Reply:
The defaults are indeed both the configured value, which is actually a rubin-env version.